frontend/public/components/utils/_log-window.scss

```diff
@@ -1,4 +1,6 @@
-// vars
+// TODO These variables should be in the main 'frontend/public/styles/_vars.scss' file, but there is a bug that causes
+// Webpack to fail if pf variables are used in that file. For some reason, pf variables are undefined when `_vars.scss`
+// is compiled.
```

Comment:
Are we sure it's a webpack bug and not just a config problem?

cc @alecmerdler

Reply:
I think we had started defining variables that are used in only one file where they're used anyway.

Reply:
Hmm, okay. I wasn't sure if this was the approach we wanted, so I added this comment. I ran into this a while back and Geoff took a brief look at it. We weren't able to figure out why it wasn't working. I'm sure it's related to the Webpack config in some way, but I couldn't figure it out.

Reply:
I added a Jira issue for this, so I'll remove this comment before merging.